scvi/model/_multivi.py

```diff
@@ -865,6 +895,132 @@ def differential_expression(
 
         return result
 
+    @torch.no_grad()
```

Comment:
Nit: The preferred decorator now is `torch.inference_mode()` as per #1695. However, this might be hard since we would have to change all in-place tensor ops in this function.

Reply:
Not sure what to do about this ... maybe we can discuss it but not sure about the correct way to proceed.

Reply:
We can just leave it as is for now, I can change it later.